packages/react-core/src/demos/examples/AlertGroup/AlertGroupDemo.tsx

```diff
@@ -0,0 +1,423 @@
+      Clear
+    </DropdownItem>
+  ];
+
+  const notificationDrawer = (
+    <NotificationDrawer>
+      <NotificationDrawerHeader count={getUnreadNotificationsNumber()}>
+        <Dropdown
+          onSelect={onDropdownSelect}
+          toggle={
+            <KebabToggle
+              onToggle={isActive => onDropdownToggle('dropdown-toggle-id-0', isActive)}
+              id="dropdown-toggle-id-0"
+            />
+          }
+          isOpen={openDropdownKey === 'toggle-id-0'}
+          isPlain
+          dropdownItems={notificationDrawerActions}
+          id="notification-drawer-0"
+          position={DropdownPosition.right}
+        />
+      </NotificationDrawerHeader>
+      <NotificationDrawerBody>
+        {notifications.length !== 0 && (
+          <NotificationDrawerList>
+            {notifications.map(({ key, variant, title, srTitle, description, timestamp }) => (
+              <NotificationDrawerListItem key={key} variant={variant} isRead={isNotificationRead(key)}>
+                <NotificationDrawerListItemHeader variant={variant} title={title} srTitle={srTitle}>
+                  <Dropdown
+                    position={DropdownPosition.right}
+                    onSelect={onDropdownSelect}
+                    toggle={<KebabToggle onToggle={isActive => onDropdownToggle(key, isActive)} id={key.toString()} />}
+                    isOpen={openDropdownKey === key}
+                    isPlain
+                    dropdownItems={notificationDrawerDropdownItems(key)}
+                    id={key.toString()}
+                  />
+                </NotificationDrawerListItemHeader>
+                <NotificationDrawerListItemBody timestamp={timestamp}> {description} </NotificationDrawerListItemBody>
+              </NotificationDrawerListItem>
+            ))}
+          </NotificationDrawerList>
+        )}
+        {notifications.length === 0 && (
+          <EmptyState variant={EmptyStateVariant.full}>
+            <EmptyStateIcon icon={SearchIcon} />
+            <Title headingLevel="h2" size="lg">
+              No notifications found
+            </Title>
+            <EmptyStateBody>There are currently no notifications.</EmptyStateBody>
+          </EmptyState>
+        )}
+      </NotificationDrawerBody>
+    </NotificationDrawer>
+  );
+
+  return (
+    <DashboardWrapper
+      header={<DashboardHeader notificationBadge={notificationBadge} />}
+      mainContainerId="main-content-page-layout-default-nav"
+      notificationDrawer={notificationDrawer}
+      isNotificationDrawerExpanded={isDrawerExpanded}
+    >
+      <PageSection variant={PageSectionVariants.light}>
+        <TextContent>
+          <Text component="h1">Alert Group with Notification Drawer demo</Text>
+          <Text component="p">
+            New alerts can be added with buttons below. By default, only 3 alerts are displayed. <br />
+            The rest can be accessed in the Notification Drawer after clicking on the bell icon in the header
```

Comment:
Sentence case for "notification drawer"

Additionally:

-  this might be more of a nitpick so let me know what you think, it might  be better to remove the `<br/>` tags and have the instructions wrap normally.
- Could we also add in some verbiage about how the alerts will automatically time out here as well as in the demo description in the MD file?

Reply:
I added the `<br/>`s because the line itself seemed too long for me, but when I think about it now, maybe it'll be better to remove them because the `<br/>`s might break the text on other screen widths.